api/src/middleware/authenticate.test.ts

```diff
@@ -7,28 +7,29 @@ import env from '../env';
 import { InvalidCredentialsException } from '../exceptions';
 import { handler } from './authenticate';
 import '../../src/types/express.d.ts';
+import { vi, afterEach, test, expect } from 'vitest';
 
-jest.mock('../../src/database');
-jest.mock('../../src/env', () => ({
+vi.mock('../../src/database');
+vi.mock('../../src/env', () => ({
 	SECRET: 'test',
 }));
```

Comment:
You need to import the actual module as well, so the other env values will be available:
```suggestion
vi.mock('../../src/env', async () => {
	const actual = await vi.importActual('../../src/env');
	return {
		...(actual as object),
		SECRET: 'test',
	};
});
```